cl/api/models.py

```diff
@@ -1,10 +1,64 @@
+from enum import Enum
+
 from django.conf import settings
+from django.contrib.auth.models import User
+from django.db import models
 from django.db.models.signals import post_save
 from django.dispatch import receiver
 from rest_framework.authtoken.models import Token
 
+from cl.lib.models import AbstractDateTimeModel
+
 
 @receiver(post_save, sender=settings.AUTH_USER_MODEL)
 def create_auth_token(sender, instance=None, created=False, **kwargs):
     if created:
         Token.objects.create(user=instance)
+
+
+class WebhookEventType(Enum):
+    RECAP_EMAIL = "Recap Email"
+
+    @classmethod
+    def choices(cls):
+        return [
+            (
+                key.value,
+                key.value,
+            )
+            for key in cls
+        ]
+
+
+class Webhook(AbstractDateTimeModel):
+    user = models.ForeignKey(
+        User,
+        help_text="The user that has provisioned the webhook.",
+        related_name="webhooks",
+        on_delete=models.CASCADE,
+    )
+    event_type = models.CharField(
+        max_length=20,
+        choices=WebhookEventType.choices(),
+    )
+    url = models.URLField(max_length=2000)
+    enabled = models.BooleanField(default=False)
+    # TODO - How do we expect to version the Webhooks?
+    webhook_version = models.CharField(max_length=10)
```

Comment:
Shall we just keep it really simple and just use an integer field starting a 1 and incrementing?